app/javascript/react/containers/DonutShow.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
+import ReviewTile from '../components/ReviewTile'
+
+
+class DonutShow extends Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      donut: [],
+      shop: [],
+      reviews: []
+    }
+  }
+
+  componentDidMount(){
+    fetch(`/api/v1/shops/${this.props.params.id[0]}/donuts/${this.props.params.id[1]}`)
```

Comment:
I will allow this, but this url path _should not_ be deeply nested ideally.

Reply:
totally. we recognize this is not necessarily **clean**. would love to refactor if we have time.